model/clusters_mgmt/v1/cluster_health_state_type.model

```diff
@@ -0,0 +1,27 @@
+/*
+Copyright (c) 2020 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// ClusterHealthState indicates the health of a cluster.
+enum ClusterHealthState {
+  // Cluster is Ready and healthy.
+  Healthy   
```

Comment:
indentation still an issue :smile: 
I think you have multiple spaces here, where the others have a single tab

Reply:
maybe now?? :man_facepalming: